clone/cmd/sumdbclone/sumdbclone.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// sumdbclone is a one-shot tool for downloading entries from sum.golang.org.
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/cenkalti/backoff/v4"
+	"github.com/golang/glog"
+	"github.com/google/trillian-examples/clone/internal/cloner"
+	"github.com/google/trillian-examples/clone/internal/verify"
+	"github.com/google/trillian-examples/clone/logdb"
+	sdbclient "github.com/google/trillian-examples/sumdbaudit/client"
+	"github.com/google/trillian/merkle/rfc6962"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var (
+	height         = flag.Int("h", 8, "tile height")
```

Comment:
More verbose flag names and descriptions for these first two would probably be good for folks jumping in.

Reply:
Made `height` a constant for now because realistically anyone changing this should know what they are doing. Added something for the vkey.